src-tauri/src/app/api.rs

```diff
@@ -86,10 +91,34 @@ impl ApiEndpoints {
     }
 
     //habe das angelegt weil in javascript wurde es schon geändert aber hier ist noch anderer wert?
-    pub async fn request_from_norisk_endpoint_with_experimental<T: DeserializeOwned>(endpoint: &str, is_experimental: bool) -> Result<T> {
+    pub async fn request_from_norisk_endpoint_with_experimental<T: DeserializeOwned>(endpoint: &str, is_experimental: bool, norisk_token: &str) -> Result<T> {
         let url = format!("{}/{}/{}", get_launcher_api_base(is_experimental), NORISK_LAUNCHER_API_VERSION, endpoint);
         println!("URL: {}", url); // Den formatierten String ausgeben
         Ok(HTTP_CLIENT.get(url)
+            .header("Authorization", format!("Bearer {}", norisk_token))
+            .send().await?
+            .error_for_status()?
+            .json::<T>()
+            .await?
+        )
+    }
+
+    pub async fn post_from_norisk_endpoint_with_experimental<T: DeserializeOwned>(endpoint: &str, is_experimental: bool, norisk_token: &str) -> Result<T> {
+        let url = format!("{}/{}/{}", get_launcher_api_base(is_experimental), "api/v1", endpoint);
+        println!("URL: {}", url); // Den formatierten String ausgeben
```

Comment:
Please change to debug log

Reply:
das war schon, kommt jz nt in den pr

Reply:
habe die schlimmsten sachen refactored, aber das juckt und gehört hier nt rein